File: Java/_058_Length_of_Last_Word/Solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count = 0;
        int start = s.length() - 1;
        
        if (s.length() <= 0)
            return 0;      
        else 
        {
            int k, i;
            for (k = 1; k < s.length() + 1; k++) 
            {
                if (s.charAt(s.length() - k) == ' ')
                    start -= 1;
                else
                    break;
            }
                    
            if (start == -1)
                return 0;
            
            for (i = k; i < s.length() + 1; i++) 
            {
                if (s.charAt(s.length() - i) != ' ')
                    count += 1;
                else
                    break;
            }

            return count;
        }
    }
}

File: Python/_058_Length_of_Last_Word/solution.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        start = len(s) - 1
        
        if len(s) <= 0:
            return 0
        
        else:
            for k in range(1, len(s) + 1):
                if s[len(s) - k] == ' ':
                    start -= 1
                else:
                    break
                    
            if start == -1:
                return 0
            
            for i in range(k, len(s) + 1):
                if s[len(s) - i] != ' ':
                    count += 1
                else:
                    break

            return count        
